src/common/config/state/init.js

```diff
@@ -8,7 +8,6 @@ const model = () => {
     maskInputOptions: { password: true }
   }
   return {
-    allow_bfcache: true, // *cli - temporary feature flag for BFCache work
```

Comment:
does the agent fail or warn or anything negative if a customer is setting this value via a previous version implementation and we remove it from the config model?

Reply:
The set fn just ignores it via the configurable.js